### PR TITLE
Enable Source Link

### PR DESCRIPTION
**Pull request by chuckries**

### CODE REVIEW

File: src/MIDebugEngine.sln [hunk 1]
```diff
@@ -41,6 +41,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Android", "Android", "{2865
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{CF02407C-BF37-4D51-83F4-845A7F36C101}"
 	ProjectSection(SolutionItems) = preProject
+		Directory.Build.props = Directory.Build.props
```
Comment (gregg-miskelly) [inline src/MIDebugEngine.sln]: Remove?

**Comment by gregg-miskelly** [comment]
Otherwise LGTM

File: src/MIDebugEngine.sln (hunk 1 above)
Comment (chuckries) [reply src/MIDebugEngine.sln]: thanks.

**Comment by pieandcakes** [comment]
@chuckries asked me to push it.

**Pull request merged**